controllers/messageController.js

```diff
@@ -29,7 +29,7 @@ const sendMessage = async( req, res) => {
 }
 const updateMessage = async( req, res) => {
     try {
-        if( !req?.body?.id) return res.status(400).json({"message":"ID not present."});
+        if( !req?.body?.id || !mongoose.Types.ObjectId.isValid(req.body.id)) return res.status(400).json({"message":"ID not present."});
```

Comment:
Can you use Jsend format in your response.

Reply:
ok, I will.